phpcs.xml.dist

```diff
@@ -21,11 +21,33 @@
         <exclude name="SlevomatCodingStandard.Classes.DisallowLateStaticBindingForConstants.DisallowedLateStaticBindingForConstant"/>
     </rule>
 
-    <rule ref="SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingParameterTypeHint">
+    <!-- See https://github.com/squizlabs/PHP_CodeSniffer/issues/2937 -->
+    <rule ref="Squiz.Arrays.ArrayDeclaration.ValueNoNewline">
```

Comment:
Should this be just excluded from the ruleset instead of being disabled for every path individually?